### ISSUE TITLE
[Feature request] Add the option to change the QoS of a topic

### ISSUE THREAD

**Issue by charlielito** [dheera/rosboard#93]
When QoS doesn't match, nothing is received. For that an option to set up the QoS of the topic like in rviz2 would be very helful. For example for `OccupancyGrid` topics for static maps, normally the QoS is `Durability: TRANSIENT_LOCAL`, which currently doesn't work with roasboard.

**Comment by dheera** [comment]
@charlielito :Would here be any major issues if we set the policy to all topics as TRANSIENT_LOCAL by default, before such a selection exists?

Actually I wasn't even able to get TRANSIENT_LOCATION working properly even with publisher and subscriber matching:

https://answers.ros.org/question/382626/latching-doesnt-work-in-ros2/

**Comment by charlielito** [comment]
I don't think that would be a proper solution, since most of topics are `VOLATILE`. According to this [link](https://docs.ros.org/en/rolling/Concepts/About-Quality-of-Service-Settings.html#:~:text=QoS%20profile%20compatibility%20is%20determined,it%20is%20able%20to%20provide.), `VOLATILE` publishers are not compatible with `TRANSIENT_LOCAL` subscriptions.

In my case, I've been working with ROS2 foxy and galactic and it __does__ work `TRANSIENT_LOCAL` publisher and subscribers.

**Comment by rscova** [comment]
Hi @dheera, I am having issues seeing RBG cameras in the dashboard, I think it is because the Reliability Policy is "Reliable" instead of "Best Effort" which is how Gazebo 11 publishes this topic.

**Comment by dheera** [comment]
Hi folks,
Sorry I haven't been active on this for a while due to a new job. Let me look into this issue and try to reproduce all the combinations of QoS to see what works.
